loader/loader.c

```diff
@@ -1803,6 +1803,10 @@ void loader_initialize(void) {
 #if defined(GIT_BRANCH_NAME) && defined(GIT_TAG_INFO)
     loader_log(NULL, VULKAN_LOADER_INFO_BIT, 0, "[Vulkan Loader Git - Tag: " GIT_BRANCH_NAME ", Branch/Commit: " GIT_TAG_INFO "]");
 #endif
+
+#if defined(LOADER_DISABLE_DYNAMIC_LIBRARY_UNLOADING)
+    loader_log(NULL, VULKAN_LOADER_WARN_BIT, 0, "Vulkan Loader: library unloading is disabled");
```

Comment:
Not sure `vulkan_loader_debug_flags` is the most appropriate for this.

Reply:
Warn is appropriate. Lets people know that it is happening, and if they didn't intend it, will be a good reminder.

Reply:
Oh I just realized where this log message is (initialization) and I'm not sure it'll be seen by people.

But I don't know of a better place, save for during vkDestroyInstance, for this to be. Maybe it should be in both locations.

Reply:
On second thought, there really is not a better place. Plus, the parallels with the git version info makes a lot of sense to keep together.